torch/_dynamo/skipfiles.py

```diff
@@ -171,8 +171,6 @@ def check(filename, allow_torch=False):
 
 # skip common third party libs
 for _name in (
-    "einops",
-    "einops_exts",
```

Comment:
Can you check if `einops_exts` is still needed for:
```
./benchmarks/dynamo/torchbench.py --performance --inference -k DALLE2_pytorch
```

Reply:
I suspect https://github.com/pytorch/pytorch/pull/99207 will actually fix that issue, so maybe we can remove `einops_exts` and land the `ConstantVariable` part of #99207.

Reply:
sure. i guess up to you guys, if we land this first i can do a follow up once 99207 lands. or i can just wait on that and rebase this

Reply:
We can land this.  `DALLE2_pytorch` isn't working in inductor anyway, so it won't move the stats.

Reply:
> is it useful for me to land this PR and keep einops_ext skipped?

@wconstab  yes, it is useful 

`einops_ext` is a separate module with additional functions by @lucidrains , so maybe he can speak to the best way forward
(I am not sure @-tagging , I can ping him via email)

Reply:
ok, well, we ended up leaving einops_ext un-skipped and hopefully will be landing a fix later that addresses string-formatting issues that broke to_/from_ ops there.  If we come up with additional issues later we can address them then.

Reply:
@arogozhnikov hey yes! that package predates your addition of `pack` and `unpack`

i've removed it from both dalle2-pytorch and imagen-pytorch this morning

do let me know if you find any more traces of it in other packages